src/test/scala/WikiTextHelperTest.scala

```diff
@@ -75,4 +75,20 @@ class WikiTextHelperTest extends FunSuite {
     val expected = "The/O quick/B-ORG brown/I-ORG fox/I-ORG jumps/O over/O the/O lazy/B-PER dog/I-PER ./O Sometimes/O life/B-ORG is/O going/O to/O hit/O you/O in/O the/O head/B-PER with/I-PER a/I-PER brick/I-PER ./O Don't/O lose/O faith/B-MISC ./O"
     assert(annotated.equals(expected))
   }
+
```

Comment:
I guess should update test in line 44 for `splitEntity` new changes.

Reply:
Yeah, agreeed.